__tests__/index.test.js

```diff
@@ -105,7 +87,64 @@ describe('/api/reviews/:review_id', () => {
             .get("/api/reviews/777")
             .expect(404)
             .then(({ body }) => {
-                expect(body.msg).toBe("Review 777 could not be found");
+                expect(body.msg).toBe("Not Found");
+            });
+    });
+});
+
+
+describe('GET /api/reviews/:review_id/comments', () => {
+    test('GET:200 responds with an array of comments for the given review_id of which each comment should have the following properties:- `comment_id`- `votes`- `created_at`- `author` which is the `username` from the users table- `body`- `review_id`comments should be served with the most recent comments first',
+        () => {
+            return request(app)
+                .get('/api/reviews/2/comments')
+                .expect(200)
+                .then((response) => {
+                    const { comments } = response.body;
+                    expect(comments.length > 0).toBe(true);
+                    comments.forEach(comment => {
+                        expect(comment).toMatchObject({
+                            comment_id: expect.any(Number),
```

Comment:
Although it's covered by your next expect, you should be asserting against review_id here.